src/index.ts

```diff
@@ -72,15 +75,17 @@ export class Resolver {
    * @returns {Promise<AccountId>}
    */
   public async resolveSLD(domain: string): Promise<string | undefined> {
+    const isUnstoppableDomain = await this._unstoppableDomainsResolver.isSupportedDomain(domain);
```

Comment:
it is possible for this to be undefined since it is not defined in the constructor but inside the `init()` function. We should verify it exists before using it

Reply:
Oops let me use more speific language: 

it is possible for `this._unstoppableDomainsResolver` to be undefined since it is not defined in the constructor but inside the init() function. We should verify it exists before using it